components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx

```diff
@@ -47,18 +49,79 @@ const KeyManagerNonceChecker: React.FC = () => {
   return (
     <div className="container mt-5">
       <h2 className="title is-2">Nonce</h2>
-      <p>
-        Retrieve the nonce of an <code>address</code> for a specific Key
-        Manager.
-      </p>
+      <article className="message is-info">
+        <div className="message-body">
+          This tool will retrieve the nonce of a
+          <a
+            href="https://docs.lukso.tech/standards/universal-profile/lsp0-erc725account"
+            target="_blank"
+            rel="noreferrer"
+            className="ml-1"
```

Comment:
Why does any single link a tag needs a custom ml or mt ? It sounds off to me.

Not a big deal but I don't understand why

Reply:
Same as on the other repository:

Its either `mx-1` on every **mid-sencente**-link or `{' '}` behind AND in front of the element. Class looks much nicer and failsafe imo. Also saw it in the examples of Bulma.

Reply:
I reco to stick with `{' '}` let's not create fake spaces by using margin. Let's try to stick to a good semantic.